### PR TITLE
Fix undesirable separation when hiding first element in Container

### PR DESCRIPTION
**Pull request by dclaux**
Fix lingering visualizer bug introduced by previous change in renderer

### CODE REVIEW

File: source/nodejs/adaptivecards/src/card-elements.ts [hunk 1]
```diff
@@ -118,7 +118,12 @@ export abstract class CardElement {
         }
 
         if (this._separatorElement) {
-            this._separatorElement.style.display = this._isVisible ? this._defaultRenderedElementDisplayMode : "none";
+            if (this.parent && this.parent.isFirstElement(this)) {
+                this._separatorElement.style.display = "none";                
+            }
+            else {
+                this._separatorElement.style.display = this._isVisible ? this._defaultRenderedElementDisplayMode : "none";
+            }
```
Comment (tongyliu) [inline source/nodejs/adaptivecards/src/card-elements.ts]: Could simplify into one conditional:

```
let shouldShowSeparator = this._isVisible && !(this.parent && this.parent.isFirstElement(this));
```
(or something similar).

File: source/nodejs/adaptivecards/src/card-elements.ts [hunk 2]
```diff
@@ -2879,7 +2885,7 @@ export class Container extends CardElement {
         for (let item of this._items) {
             handleElement(item);
         }
-
+
```
Comment (tongyliu) [inline source/nodejs/adaptivecards/src/card-elements.ts]: Nit: unnecessary whitespace

File: source/nodejs/adaptivecards/src/card-elements.ts (hunk 1 above)
Comment (dclaux) [reply source/nodejs/adaptivecards/src/card-elements.ts]: Actually I think that would make it harder to read and understand, which is why I did it in a more explicit way.

File: source/nodejs/adaptivecards/src/card-elements.ts (hunk 2 above)
Comment (dclaux) [reply source/nodejs/adaptivecards/src/card-elements.ts]: This is followed by "return true". There should always be a blank line after a closing curly bracket unless there is another closing curly bracket on the next line or if there is an else statement.

Comment (tongyliu) [reply source/nodejs/adaptivecards/src/card-elements.ts]: Oh I was referring to the extra spaces that got added to the blank line (darker green highlighted section), not the line itself.

Comment (dclaux) [reply source/nodejs/adaptivecards/src/card-elements.ts]: Oh! Sorry :)
